prisma/schema.prisma

```diff
@@ -11,22 +11,23 @@ generator client {
 }
 
 model Transfer {
-  id           String         @id @default(auto()) @map("_id") @db.ObjectId
-  depositNonce Int            
-  resource     Resource?      @relation(fields: [resourceID], references: [id])
-  resourceID   String?
-  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
-  fromDomainId Int 
-  toDomain     Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
-  toDomainId   Int?
-  sender       String?
-  destination  String?
-  amount       String?
-  timestamp    DateTime?
-  status       TransferStatus
-  deposit      Deposit?
-  execution    Execution?
-  fee          Fee?
+  id              String         @id @default(auto()) @map("_id") @db.ObjectId
+  depositNonce    Int
+  resource        Resource?      @relation(fields: [resourceID], references: [id])
+  resourceID      String?
+  fromDomain      Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
+  fromDomainId    Int
+  toDomain        Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
+  toDomainId      Int?
+  sender          String?
+  destination     String?
+  amount          String?
+  timestamp       DateTime?
+  status          TransferStatus
+  deposit         Deposit?
+  execution       Execution?
+  fee             Fee?
+  convertedAmount Float?
```

Comment:
Can we please call this value or usdValue

Reply:
ok